### ISSUE TITLE
[Improvement]: Improve Orbital Drone Locking.

### ISSUE THREAD

**Issue by ghost** [YimMenu/YimMenu#2197]
### Which feature

When you want to lock on to the nearest player while using an orbital drone, there is a high probability that you will lock on to either an NPC or fixed objects on the map.

Even if the player is very close to you, you have to try very hard to lock onto him (I tried and failed to lock onto a player at least 7 times).

### Reason

Using this feature wastes a lot of time.

**Comment by pierrelasse** [comment]
For me it works good

**Comment by ghost** [comment]
> For me it works good

Yes, it works well sometimes, but it can be a bit unstable towards users in the vehicle or towards mk2 users.

**Comment by Tayipc38** [comment]
> Using this feature wastes a lot of time.

Some would argue playing this game wastes a lot of time.

**Comment by pierrelasse** [comment]
> > Using this feature wastes a lot of time.
> 
> Some would argue playing this game wastes a lot of time.

Very helpful

**Comment by Tayipc38** [comment]
> > > Using this feature wastes a lot of time.
> > 
> > 
> > Some would argue playing this game wastes a lot of time.
> 
> Very helpful

I made this feature so I can say whatever the fuck I want.
Besides, Itll probably be me again, to fix this.
